test/protobuf/decoder_test.exs

```diff
@@ -126,7 +126,7 @@ defmodule Protobuf.Decoder.Test do
       use Protobuf, from: Path.expand("../proto/mumble.proto", __DIR__)
     end
 
-    msg = MumbleProto.Authenticate.new(username: "bitwalker")
+    msg = MumbleProto.MumbleProto.Authenticate.new(username: "bitwalker")
```

Comment:
I think `inject` is still fine, it's just that one now has to understand that if you are using package namespaces that it will inject the namespace (assuming I understand what you are saying the change is correctly). As far as coordinating this as a breaking change, I'll probably release this as a new major version (1.x), and update the README with a list of things one has to review in order to upgrade. If you could start documenting those things as part of this change, I think that will go a long way towards easing the transition.